.github/workflows/build.yml

```diff
@@ -24,6 +121,7 @@ jobs:
           python-version: '3.9'
 
       - name: Verify PR
+        id: verify_pr
```

Comment:
this step can be moved under sanity check job, right?

Reply:
Now everything under a common job.